src/app/components_ngrx/planner-list/ngx-datatables.less

```diff
@@ -3,9 +3,18 @@
 
 .cell-class {
   overflow: visible !important;
-  padding-top: 3px !important;
-  padding-bottom: 4px !important;
-  line-height: 44px !important;
+  align-items: center;
+  height: 100%;
+  vertical-align: middle;
```

Comment:
I do not see any use of these three properties here. ```align-items: center; height: 100%; vertical-align: middle;```.

Reply:
cell items can be impacted if we don't set it to center and height 100% makes sure any item takes full height to keep everything vertically centered